cypress/support/commands.js

```diff
@@ -26,15 +26,15 @@
 
 import { db } from '../../src/lib/firebase';
 
-Cypress.Commands.add("addItem", (token, itemID, dateOfPurchase) => {
+Cypress.Commands.add("addItem", (token, itemID, dateOfPurchase, numberOfPurchases = 1, numberOfDays = 14) => {
```

Comment:
**No need to change:**

When you create a function, is always better to create an object for your function arguments (I'm sorry I didn't catch this before!)

With the current implementation, you'll have to remember the order of each argument (token first, itemID second, dateOfPurchase third...and so on). This could lead to some problems:

1. Given the case where `itemID` is null, you'd have to be explicit about that argument being null 
```js
cy.addItem('token123`, null, testDate, 2)
``` 
2. The more arguments you have, the more complicated is to read the function:

```js
cy.addItem('token123`, null, testDate, 2)

/**
Here I only _know_ that the first argument is a token, and also that the third argument is a date.
But I have no clue about what other arguments are, for example, I have no idea what should I use instead of `null`
or what should I use instead of `2`
*/
```

A better approach will be

```js
Cypress.Commands.add(
  "addItem",
  ({ token, itemID, dateOfPurchase, numberOfPurchases = 1, numberOfDays = 14}) => {...}
);
```

That will let me do call the function in a more idiomatic way:

```js
cy.addItem({
  token: 'whatEverToken',
  numberOfPurchases: 3,
  numberOfDays: 0,
  itemId: '123',
  dateOfPurchase: theDate
})
```

As you can see, we fixed the 2 problems I mentioned above:

1. The order of the arguments does not matter
2. I know what I'm really passing to the function

cc @the-collab-lab/tcl-2

Reply:
Hm I do like that a lot better - though to note, I don't think we'd ever have cause to have the `itemID` null, really. I really like the idea of not worrying about the order of the arguments though - It got a little hefty with the added parameters. 😬